[Maps] Updated documentation for Elastic Maps Server 7.13 (#98310)
elastic/kibana: docs/maps/connect-to-ems.asciidoc

  {ems-docker-image}
----------------------------------

> Once {hosted-ems} is running, follow instructions from the webpage at `localhost:8080` to define a configuration file and optionally download a more detailed basemaps database.

Reviewer: I don't see this content in the preview of the doc.

Author: There's a conditional for this part to show up for released version builds, I don't know if there's a way to see this in a preview 😓

| `port`
 | Specifies the port used by the backend server. Default: *`8080`*. <<server-port,Equivalent {kib} setting>>. 

| `basePath`

>  | Specify a path to mount the server as if you are running behind a proxy. This setting cannot end in a slash (`/`). <<server-basePath,Equivalent {kib} setting>>. 

Reviewer 2: nit: I'm not an expert but I think this prepositional phrase is more pleasing.

```suggestion
 | Specify a path at which to mount the server if you are running behind a proxy. This setting cannot end in a slash (`/`). <<server-basePath,Equivalent {kib} setting>>. 
```

[[elastic-maps-server-data]]
==== Data

> {hosted-ems} hosts vector layer boundaries and vector tile basemaps for the entire planet. Boundaries include world countries, global administrative regions, and specific country regions. A basemap up to zoom level 8 is bundled in the Docker image. This basemap is sufficient for maps and dashboards at country level, but if you want to present maps with higher detail follow the instructions of the front-page to download the appropriate basemap. The most detailed basemap at zoom level 14 is good for street level maps, and takes around 90GB of disk space.

Reviewer 2: This suggestion is based on the idea that `basemaps` in our documentation refers to the three basemap styles included with EMS, so I think we should consistently use the plural `basemaps`. And, when referring to the download, we should use a singular `database` where `basemaps` is the adjective describing the database. 

```suggestion
{hosted-ems} hosts vector layer boundaries and vector tile basemaps for the entire planet. Boundaries include world countries, global administrative regions, and specific country regions. Basemaps up to zoom level 8 are bundled in the Docker image. These basemaps are sufficient for maps and dashboards at country level. But, if you want to present maps with higher detail, follow the instructions of the front-page to download and configure the appropriate basemaps database. The most detailed basemaps at zoom level 14 are good for street level maps but require ~90GB of disk space.
```

> {hosted-ems} hosts vector layer boundaries and vector tile basemaps for the entire planet. Boundaries include world countries, global administrative regions, and specific country regions. Basemaps up to zoom level 8 are bundled in the Docker image. These basemaps are sufficient for maps and dashboards at country level. But, if you want to present maps with higher detail, follow the instructions of the front-page to download and configure the appropriate basemaps database. The most detailed basemaps at zoom level 14 are good for street level maps but require ~90GB of disk space.

Reviewer: ```suggestion
{hosted-ems} hosts vector layer boundaries and vector tile basemaps for the entire planet. Boundaries include world countries, global administrative regions, and specific country regions. Basemaps up to zoom level 8 are bundled in the Docker image. These basemaps are sufficient for maps and dashboards at the country level. To present maps with higher detail, follow the instructions of the front page to download and configure the appropriate basemaps database. The most detailed basemaps at zoom level 14 are good for street level maps, but require ~90GB of disk space.
```

Reviewer: Be more specific about what front page you are referring to

> [role="screenshot"]

Reviewer: Can this image either show the whole screen, or be cut just to show the zoom level menu?

Reviewer: I defer to Nick on `basemaps database`.

Author: I'll crop it to just the zoom level menu since we don't want to show the URL behind it :+1:

image::images/elastic-maps-server-basemaps.png[Basemaps download options]

> TIP: The available basemaps and boundaries can be explored from the `/maps` endpoint in a web page that is your self-managed equivalent to https://maps.elastic.co

Reviewer: ```suggestion
TIP: The available basemaps and boundaries can be explored from the `/maps` endpoint in a web page that is your self-managed equivalent to https://maps.elastic.co.
```
